R/util.R

```diff
@@ -74,6 +84,32 @@ ConvertibleToPythonList = function(input) {
   return(input)
 }
 
+# Helper function to call Python and convert error messages
+CallPython <- function(func, args) {
+  tryCatch(
+    expr = {
+      return(do.call(func, args))
+    },
+    error = function(err) {
+      message(paste("Error calling Python function: ", func))
+      if (str_detect(conditionMessage(err),
```

Comment:
is it possible to catch by the type of exception thrown instead of error message parsing?

Reply:
Apparently it's possible but uncommon: https://screenshot.googleplex.com/3YkHzfmZiH6 

wdyt?